swf/src/types.rs

```diff
@@ -302,6 +302,7 @@ pub struct PlaceObject<'a> {
     pub version: u8,
     pub action: PlaceObjectAction,
     pub depth: Depth,
+    pub id: Option<CharacterId>,
```

Comment:
Isn't the character ID already part of `PlaceObjectAction`? I don't have my laptop open to check right now, but I could swear that was the case...

Reply:
You were totally right. Tank you.